gis/openstreetmap-carto/Portfile

```diff
@@ -0,0 +1,203 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           active_variants 1.1
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        gravitystorm openstreetmap-carto 5.3.1 v
+github.tarball_from archive
+revision            0
+
+categories          gis
+platforms           darwin
+maintainers         {@frankdean fdsd.co.uk:frank.dean} openmaintainer
+
+# license CC0-1.0 (http://creativecommons.org/publicdomain/zero/1.0/)
+license             public-domain
+
+description         A general-purpose OpenStreetMap mapnik style, in CartoCSS
+
+long_description    These are the CartoCSS map stylesheets for the Standard \
+                    map layer on OpenStreetMap.org.
+
+checksums           rmd160  f3effe425ea8c0d54f5437f22fe0181bb1005ea3 \
+                    sha256  be63d4fc06f1db447f1a249515f9fc83449f7f3027469d1cc051150ded3f85e9 \
+                    size    2719803
+
+python.default_version 39
+
+depends_lib-append  port:py${python.version}-psycopg2 \
+                    port:py${python.version}-requests \
+                    port:py${python.version}-yaml
+
+# Database variants (from the GDAL port)
+set postgresql_suffixes {12 13}
+
+set portsgresql_variants {}
+set postgresql_default_variant "if {"
+
+foreach s ${postgresql_suffixes} {
+    lappend portsgresql_variants postgresql${s}
+    set postgresql_default_variant "${postgresql_default_variant}!\[variant_isset postgresql${s}] && "
+}
+
+set postgresql_default_variant [string range ${postgresql_default_variant} 0 end-4]
+set postgresql_default_variant "${postgresql_default_variant}} {default_variants +postgresql${s}}"
+eval $postgresql_default_variant
+
+foreach s ${postgresql_suffixes} {
+    set p postgresql${s}
+    set v [string index ${s} 0].[string index ${s} 1]
+    set i [lsearch -exact ${portsgresql_variants} ${p}]
+    set c [lreplace ${portsgresql_variants} ${i} ${i}]
+    variant ${p} description "Enable PostgreSQL ${v} support" conflicts {*}${c} "
+        depends_lib-append      port:${p}
+    "
+}
```

Comment:
I know you copied this verbatim from the `gdal` port, but if there are only two variants here I wouldn't add this amount of code to automate things that don't really need it... and besides that you are overwriting the variants you declare here below where you redefine the variants. So please remove this whole black here and update the variant descriptions below.

Reply:
Ok